packages/mos-gateway/src/coreHandler.ts

```diff
@@ -465,18 +474,21 @@ export class CoreMosDeviceHandler {
 	 * @param o the object to convert
 	 */
 	private fixMosData(o: any): any {
-		if (_.isObject(o) && (o instanceof MosTime || o instanceof MosDuration || o instanceof MosString128)) {
```

Comment:
Didnt we discus this fixMosData function and decide to remove it?

Reply:
Note to self: we did discuss it, but (I) decided to (try to) not break-everything in this PR